qa/L0_perf_nomodel/run_test.sh

```diff
@@ -81,9 +84,15 @@ for BACKEND in $BACKENDS; do
         continue
     fi
 
-    # set naming (special case for libtorch model)
-    INPUT_NAME="INPUT0" && [ $BACKEND == "libtorch" ] && INPUT_NAME="INPUT__0"
-
+    # set input name (special case for libtorch and python model)
+    if [ $BACKEND == "libtorch" ]; then
+        INPUT_NAME="INPUT__0"
+    elif [ $BACKEND == "python" ]; then
+        INPUT_NAME="IN"
```

Comment:
You can rename "IN" to "INPUT0" so that this if condition is not required.

Reply:
Solved in follow up PR. Is there a reason the python backend uses a different naming convention? I think we should keep the naming consistent i.e. INPUT0 and OUTPUT0 instead of IN and OUT

Reply:
Great, no there is not any reason. My bad, I should be more consistent in the naming of inputs and outputs for QA models.